README.zh-CN.md

```diff
@@ -61,23 +61,198 @@ Apache EventMesh提供了许多功能来帮助用户实现他们的目标，以
 - [EventMesh-catalog](https://github.com/apache/eventmesh-catalog): 使用 AsyncAPI 进行事件模式管理的目录服务。
 - [EventMesh-go](https://github.com/apache/eventmesh-go): EventMesh 运行时的 Go 语言实现。
 
-## 快速开始
+## 快速入门   
 
-指南:
+本节指南将指导您分别从[本地](#在本地运行-eventmesh-runtime)、[docker](#在-docker-中运行-eventmesh-runtime-)、[k8s](#在-kubernetes-中运行-eventmesh-runtime-)部署EventMesh的步骤:
+
+本节指南只是帮助您快速入门EventMesh部署，按照默认配置启动EventMesh，如果您需要更加详细的EventMesh部署步骤，请访问[EventMesh官方文档](https://eventmesh.apache.org/docs/next/introduction)。
+
+### 部署 EventMesh Store  
+
+> EventMesh现在支持`standalone`、`RocketMQ`、`Kafka`等中间件作为存储   
+> 如果是在非`standalone`模式下，需要先部署所需的`store`，以`rocketmq`模式为例: 部署[RocketMQ](https://rocketmq.apache.org/docs/quickStart/01quickstart/)
```

Comment:
You may add a hyperlink displaying how many Event Store are supported like https://eventmesh.apache.org/zh/docs/instruction/runtime#13-配置:

![image](https://github.com/apache/eventmesh/assets/41445332/ed552252-41bf-4ed4-af8e-9003aed9e168)

Connectors also have a sheet like this.

Besides, please use Event Store instead of `store` to clarify concept.